package.json

```diff
@@ -94,7 +96,7 @@
     "@types/toastify-js": "^1.11.1",
     "@typescript-eslint/eslint-plugin": "^5.59.5",
     "@typescript-eslint/parser": "^5.59.5",
-    "bootswatch": "^5.2.3",
+    "bootstrap-v4": "npm:bootstrap@^4.6.2",
```

Comment:
The version of bootstrap listed in the prod dependencies is 5.2.3. Should we be using v4 here?

Reply:
Yeah, that's a bit weird. Version 5 seems to be introduced in 15eb8326 as part of regular dependency updates. But they are not compatible and there's quite a few breaking changes (I tried compiling the themes with BS5 as part of this MR and the design was completely broken...).

But we'll have to check if any of our newer code depends on BS5 features. We use two JS components (collapse and dropdown). I think I'd rather get rid of the BS5 dependency in a separate pull request to avoid making this one even more confusing.